README.md

```diff
@@ -32,6 +32,8 @@ Then access http://localhost:8080/
 - **Ben Shen** - Developer
 - **Yuxuan Chen** - Designer
 - **Cristie Huang** - Designer
+- **Kehui Guo** - Designer
+- **Miranda Yu** - Business
```

Comment:
Kathleen and Brandon are listed as `PMM` instead of `Business` last year if we want them to be the same!